admin/complex-mechanics/pyramid.md

```diff
@@ -0,0 +1,15 @@
+---
+sidebar_position: 2
+---
+
+# Pyramid
+
+Pyramid is a gambling mechanic based on Ponzi scheme. 
+
+## Deployment
```

Comment:
Staking and pyramid is not mean to be deployed on B2C version, please this section from two pages

Reply:
For this need a little bit more information